packages/RNTester/package.json

```diff
@@ -12,11 +12,9 @@
   "dependencies": {
     "@react-native-community/async-storage": "^1.11.0",
     "@react-native-community/cli-platform-android": "^4.10.1",
-    "@react-native-community/datetimepicker": "^2.4.2",
     "@react-native-community/picker": "^1.6.3",
     "@react-native-community/progress-bar-android": "^1.0.3",
     "@react-native-community/progress-view": "^1.0.3",
-    "@react-native-community/push-notification-ios": "^1.2.2",
```

Comment:
@AnshG714 i was referring to these actually

Reply:
@YashKumarVerma These are not used anywhere else, hence I removed them.